style.css

```diff
@@ -0,0 +1,414 @@
+}
+
+.header {
+  display: flex;
+  align-items: center;
+}
+
+.header::before {
+  content: '';
+  position: absolute;
+  width: 461px;
+  height: 1800px;
+  background: #000;
+}
+
+.header__title {
+  position: relative;
+  width: 98px;
+  height: 37px;
+  margin: 36px 0 0 52px;
+  font-family: Montserrat, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 30px;
+  line-height: 37px;
+  text-align: center;
+  letter-spacing: .07em;
+  color: #fff;
+}
+
+.header__language {
+  position: relative;
+  display: flex;
+  width: 238px;
+  justify-content: flex-end;
+  align-items: center;
+  top: 15px;
+}
+
+.header__language-ru {
+  color: #fff;
+  background: url('img/lang__switch.png') right no-repeat;
+  padding: 0 18px 0 0;
+  font-size: 20px;
+  line-height: 24px;
+  height: 25px;
+  width: 31px;
+}
+
+.header__language-ru:: after {
+  content: "";
+  color: #fff;
+  position: absolute;
+}
+
+.header__language-ru: hover {
+  transform: scale(1.3);
+}
+
+.header__menu {
+  display: flex;
+  margin-left: auto;
+  position: relative;
+  align-self: flex-end;
+  width: 70px;
+  height: 70px;
+  background: #f5f5f3;
+  cursor: pointer;
+}
+
+.header__menu: hover {
+  background: #696969;
+}
+
+.inst {
+  background: #fff url('img/inst_cir.png') center no-repeat;
+  width: 43px;
+  height: 43px;
+  border-radius: 50%;
+  left: 52px;
+  top: 816px;
+  display: inline-block;
+}
+
+.inst:hover {
+  background: linear-gradient(to bottom, #833ab4, #c13584, #f77737, #ffdc80);
+}
+
+.vk {
+  background: #fff url('img/vk.png') center no-repeat;
+  width: 43px;
+  height: 43px;
+  border-radius: 50%;
+  left: 107px;
+  top: 816px;
+  display: inline-block;
+}
+
+.vk:hover {
+  background: #5b88bd url('img/vk.png') center no-repeat;
+}
+
+.fb {
+  background: #fff url('img/fb.png') center no-repeat;
+  width: 43px;
+  height: 43px;
+  border-radius: 50%;
+  left: 162px;
+  top: 816px;
+  display: inline-block;
+}
+
+.fb:hover {
+  background: #4267b2 url('img/fb.png') center no-repeat;
+}
+
+.block {
+  position: absolute;
+  top: 168px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  max-width: 1500px;
+}
+
+.menu__item {
+  list-style-type: none;
+  padding: 0;
+  margin: auto;
+}
+
+#menu__toggle {
+  display: none;
+}
+
+.header__burger {
+  right: 0;
+  margin-left: auto;
+  display: flex;
+  position: absolute;
+  width: 92px;
+  height: 92px;
+  background-color: #ededed;
+}
+
+.header__burger > span,
+.header__burger > span::after {
+  display: block;
+  position: absolute;
+  width: 34.7px;
+  height: 3.48px;
+  margin-left: 28px;
+  top: 39px;
+  background-color: #000;
+}
+
+.header__burger > span::after {
+  content: '';
+  margin-left: 0;
+  top: 6px;
+}
+
+  @media (min-width: 992px) {
+    .header__menu {
+      display: flex;
+      min-height: 92px;
+      min-width: 92px;
+    }
+    .header__menu:hover {
+        background: #696969;
+      }
+  }
+
+  .menu__link {
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Locc/pullRequest?prid=5986341)